registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/RegistrationCheckerCompilerPlugin.scala

```diff
@@ -51,7 +54,10 @@ class RegistrationCheckerCompilerPlugin(override val global: Global) extends Plu
         false
     }
   }
-  override val optionsHelp: Option[String] = Some("directory where cache file will be saved")
+  override val optionsHelp: Option[String] = Some("""
+      |. - directory where cache file will be saved, required
+      |--disable - disables the plugin
```

Comment:
While you're at this change... could you add the relevant section about this compiler plugin to README.md?

Reply:
I prefer to do it as part of #26, in the later PR.

Reply:
Okay, added a checkbox there so that it's not lost